pages/star-search/index.tsx

```diff
@@ -634,7 +633,11 @@ function Chatbox({ message, userId }: { message: StarSearchChat; userId?: number
   if (typeof message.content == "string") {
     // Breaking all words so that the rendered markdown doesn't overflow the container
     // in certain cases where the content is a long string.
-    content = <Markdown className="break-all">{message.content}</Markdown>;
+    content = (
+      <Markdown remarkPlugins={[remarkGfm]} className="break-all">
```

Comment:
Need to change `break-all` to `break-words` to fix overflow that is seen below: 
![image](https://github.com/open-sauced/app/assets/20603494/9eae0a33-2c61-459f-b2ff-5adaa313cd70)